src/actions/users.js

```diff
@@ -241,7 +241,9 @@ export function getProfiles(page = 0, perPage = General.PROFILE_CHUNK_SIZE) {
         let profiles;
         try {
             profiles = await Client4.getProfiles(page, perPage);
-            removeUserFromList(currentUserId, profiles);
+            if (skipCurrent) {
+                removeUserFromList(currentUserId, profiles);
+            }
```

Comment:
Looking at your webapp code, is this change really needed? The redux store should already have the current user and you're not using the direct action result anywhere you're just selectors to pull from the store, which is guaranteed to already have the current user.

Let me know if I'm missing something

Reply:
tested, dont need this change here, closing...